src/modules/Crowbar.rb

```diff
@@ -58,6 +58,20 @@ def main
       # repos map
       @repos = {}
 
+      # Paths to repositories on SMT server
+      @smt_repos = {
+        "sles12-pool"                   => "repo/SUSE/Products/SLE-SERVER/12/%s/product",
+        "sles12-updates"                => "repo/SUSE/Updates/SLE-SERVER/12/%s/update",
+        "sles12-sp1-pool"               => "repo/SUSE/Products/SLE-SERVER/12-SP1/%s/product",
+        "sles12-sp1-updates"            => "repo/SUSE/Updates/SLE-SERVER/12-SP1/%s/update",
+        "suse-openstack-cloud-6-pool"   => "repo/SUSE/Products/OpenStack-Cloud/6/%s/product",
+        "suse-openstack-cloud-6-updates" => "repo/SUSE/Updates/OpenStack-Cloud/6/%s/update",
+        "sle12-sp1-ha-pool"             => "repo/SUSE/Products/SLE-HA/12-SP1/%s/product",
+        "sle12-sp1-ha-updates"          => "repo/SUSE/Updates/SLE-HA/12-SP1/%s/update",
+        "suse-enterprise-storage-2-pool" => "repo/SUSE/Products/Storage/2/%s/product",
+        "suse-enterprise-storage-2-updates" => "repo/SUSE/Updates/Storage/2/%s/update"
+      }
```

Comment:
From my POV, this is a configuration that definitely belongs to a configuration file. It's not a good idea to have it within the source code.